content/docs/17-glossary-all.md

```diff
@@ -16,6 +16,12 @@ import Tooltip from "shared/components/ui/Tooltip";
 # Glossary
 
 This page gives a quick reference to various object types and concepts within the Palette platform.
+## App Mode
+A mode optimized for a simpler and streamlined developer experience that allows you to only focus on the building, maintenance, testing, deployment, and monitoring of your Kubernetes applications. App Mode removes the need to worry about the infrastructure management of a Kubernetes cluster and results in a PaaS-like experience, enabling you to only focus on deploying [App Profiles](#app-profile). App Mode also allows you to deploy [Palette Virtual Clusters](#palette-virtual-cluster).

```

Comment:
the word "only" feels unnecessary to me "enabling you to ONLY focus on deploying ...", could just say "enabling you to focus on deploying ..."